apps/next-evans/components/core/PillDropdownDropdown.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Accordion } from '@reach/accordion';
+import cx from 'classnames';
+// import './PillDropdownDropdown.css';
+
+interface Props {
+  className?: string;
+  accordion?: boolean;
+  children: React.ReactNode;
+}
+
+const PillDropdownDropdown: React.FC<Props> = ({ className, accordion, children }) => {
+  const Element = accordion ? Accordion : `div`;
+  return (
+    <Element
+      className={cx(
+        className,
+        `relative rounded-lg bg-flgray-100 shadow-direct after:absolute after:top-0 after:left-[50%] after:w-0 after:h-0 after:[border:10px_solid_transparent] after:[border-bottom-color:flgray-100] after:border-t-0 after:ml-[-10px] after:mt-[-10px]`,
```

Comment:
since we're already using classnames, i like to group related utilities on separate lines as new arguments, like below. i think pulling out all of the `after:x` stuff into it's own line makes a little sense.